### PR TITLE
Add showcase page

### PR DESCRIPTION
**Pull request by trishalim**
This features Learn Anything and Invoice Radar on a `/showcase` page

### CODE REVIEW

File: homepage/homepage/app/showcase/page.tsx [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+export const metadata = {
+    title: "jazz - Showcase",
+    description: "A list of cool products built with Jazz.",
+};
+
+export default function Page() {
+    return (
+        <div className="container flex flex-col gap-6 pb-10 lg:pb-20">
+            <HeroHeader
+                title="Cool products built with Jazz"
```
Comment (aeplay) [inline homepage/homepage/app/showcase/page.tsx]:
```suggestion
                title="Built with Jazz"
```

File: homepage/homepage/app/showcase/page.tsx [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+                slogan="Find inspiration in what others have made."
```
Comment (aeplay) [inline homepage/homepage/app/showcase/page.tsx]:
```suggestion
                slogan="Great apps by smart people."
```

File: homepage/homepage/app/showcase/page.tsx [hunk 3]
```diff
@@ -0,0 +1,44 @@
+import { HeroHeader } from "gcmp-design-system/src/app/components/molecules/HeroHeader";
+import Image from "next/image";
+import { products } from "@/lib/showcase";
+import Link from "next/link";
+
+export const metadata = {
+    title: "jazz - Showcase",
```
Comment (aeplay) [inline homepage/homepage/app/showcase/page.tsx]:
```suggestion
    title: "jazz - Built With Jazz",
```

File: homepage/homepage/components/nav.tsx [hunk 4]
```diff
@@ -14,6 +14,10 @@ export function JazzNav() {
                     title: "Docs",
                     href: "/docs",
                 },
+                {
+                    title: "Showcase",
```
Comment (aeplay) [inline homepage/homepage/components/nav.tsx]:
```suggestion
                    title: "Built With Jazz",
```

File: homepage/homepage/lib/showcase.ts [hunk 5]
```diff
@@ -0,0 +1,15 @@
...
+        imageUrl: "/learn-anything.png",
+        url: "https://learn-anything.xyz",
+        description:
+            "A free, comprehensive platform that connects you to the best resources for every subject.",
+    },
+    {
+        name: "Invoice Radar",
+        imageUrl: "/invoice-radar.png",
+        url: "https://invoiceradar.com",
+        description: "Automatically gather invoices from mail and the web.",
```
Comment (aeplay) [inline homepage/homepage/lib/showcase.ts]:
```suggestion
        description: "Automatically gather invoices from mail and cloud providers.",
```

File: homepage/homepage/lib/showcase.ts [hunk 6]
```diff
@@ -0,0 +1,15 @@
+export const products = [
+    {
+        name: "Learn Anything",
+        imageUrl: "/learn-anything.png",
+        url: "https://learn-anything.xyz",
+        description:
+            "A free, comprehensive platform that connects you to the best resources for every subject.",
```
Comment (aeplay) [inline homepage/homepage/lib/showcase.ts]:
```suggestion
            "A free, comprehensive learning platform where people collect the best resources for every subject.",
```

**Review by aeplay — CHANGES_REQUESTED** [review]
This is a great MVP for this page, added some suggestions

File: homepage/homepage/components/nav.tsx (hunk 4 above)
Comment (trishalim) [reply homepage/homepage/components/nav.tsx]: Not sure about this. Looks kinda weird?

image

Comment (trishalim) [reply homepage/homepage/components/nav.tsx]: Ok nvm it's more obvious this way, but I would write it sentence case: "Built with Jazz"

**Pull request merged**